### PR TITLE
Update descriptions based on NL log triage

### PR DESCRIPTION
**Pull request by lucy-kind**
image
image

### CODE REVIEW

**Comment by jehangiramjad** [comment]
Please note that your changes to the spreadsheet already got built and merged with this PR: https://github.com/datacommonsorg/website/pull/2485 So the only file change you should keep in this PR is the one about topics. The embeddings and model.yaml files don't need to be changed (so you can undo those). After undoing those, you can git merge from latest dc master and you should be good:

1. git checkout master
2. git remote add dc https://github.com/datacommonsorg/website.git
3. git pull dc master
4. git checkout 2444-nl-tracking-bug-for-alternate-sv-description-improvements
5. git merge master
6. <hopefully there are no merge conflicts to resolve--if there are, please keep the changes from the MASTER and not your changes to the model.yaml and the csv file under the embeddings folder>
7. git push -u origin 2444-nl-tracking-bug-for-alternate-sv-description-improvements

File: deploy/base/model.yaml [hunk 1]
```diff
@@ -1 +1 @@
-embeddings_file: embeddings_us_filtered_2023_03_23_17_22_22.csv
\ No newline at end of file
+embeddings_file: embeddings_us_filtered_2023_03_24_14_26_08.csv
```
Comment (pradh) [inline deploy/base/model.yaml:1]: Could you regenerate it please?  The current head is at `embeddings_us_filtered_2023_03_25_12_09_56`, we want to make sure your timestamp is after (so it includes change I made)

**Review by pradh — COMMENTED** [review]
"what are the wealthy neighborhoods in" ... should this be a topic instead?

There's also Median_Income_Household, Median_Income_Person

File: deploy/base/model.yaml (hunk 1 above)
Comment (lucy-kind) [reply deploy/base/model.yaml]: updated embeddings. I was going on what was written on the issue bug, but can make a topic in follow up PR for the wealthy query.

Comment (pradh) [reply deploy/base/model.yaml]: Sorry, I'd consider the initial bug description as a report on the issue.  Couple of reasons for not tying "wealthy neighborhoods" to Mean_Income_Person, and instead creating a topic:
1. The data coverage for Median_Income_Household/Person is better: https://datacommons.org/tools/statvar#sv=Median_Income_Household vs. https://datacommons.org/tools/statvar#sv=Mean_Income_Household
2. Wealthy neighborhoods could be based on more factors/SVs too like home price.

And maybe we could call the topic "community affluence" or something (wealthy is one part of the spectrum)?

If you want to submit this PR though, maybe remove that one description and submit?

**Comment by lucy-kind** [comment]
sg, will remove for now and have the new topic as part of the next PR that
introduces other new topics!

**Pull request merged**